lib/adyen/rest.rb

```diff
@@ -26,11 +26,7 @@ module REST
     # @return [Adyen::REST::Client] A configured client instance
     # @see .session
     def self.client
-      Adyen::REST::Client.new(
-        Adyen.configuration.environment,
-        Adyen.configuration.api_username,
-        Adyen.configuration.api_password
-      )
+      Adyen::REST::Client.new(Adyen.configuration.environment)
```

Comment:
Should I consider adding other aliases like `Adyen::REST.payout_store_client` & `Adyen::REST.payout_review_client` ?